tensorboard/webapp/hparams/_redux/hparams_reducers_test.ts

```diff
@@ -630,4 +630,72 @@ describe('hparams/_redux/hparams_reducers_test', () => {
       expect(state2.dashboardSessionGroups).toEqual([mockSessionGroup]);
     });
   });
+
+  describe('dashboardHparamFilterAdded', () => {
+    it('adds entry dashboardFilters', () => {
+      const state = buildHparamsState();
+      const state2 = reducers(
+        state,
+        actions.dashboardHparamFilterAdded({
+          name: 'hparam1',
+          filter: {
+            type: DomainType.DISCRETE,
+            includeUndefined: true,
+            filterValues: [5],
+            possibleValues: [5, 7, 8],
+          },
+        })
+      );
+      expect(state2.dashboardFilters).toEqual({
+        hparams: new Map([
+          [
+            'hparam1',
+            {
+              type: DomainType.DISCRETE,
+              includeUndefined: true,
+              filterValues: [5],
+              possibleValues: [5, 7, 8],
+            },
+          ],
+        ]),
+        metrics: new Map(),
+      });
+    });
+  });
+
+  describe('dashboardMetricFilterAdded', () => {
+    it('adds entry dashboardFilters', () => {
+      const state = buildHparamsState();
```

Comment:
Similar: Can we add another test that contains existing items in `metrics` and ensures they are not overwritten?